### ISSUE TITLE
Failed to parse: proxyip:port at 'Register User' step (with Vagrant) 

### ISSUE THREAD

**Issue by gpolyn** [deis/deis#3045]
When I `deis register http://deis.local3.deisapp.com` and complete my data with your Vagrant-provisioned cluster, I get:
```
Traceback (most recent call last):
  File "<string>", line 2122, in <module>
  File "<string>", line 2118, in main
  File "<string>", line 2061, in _dispatch_cmd
  File "<string>", line 740, in auth_register
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.sessions", line 504, in post
  File "<string>", line 159, in request
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.sessions", line 461, in request
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.sessions", line 573, in send
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.adapters", line 337, in send
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.adapters", line 245, in get_connection
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.adapters", line 155, in proxy_manager_for
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.packages.urllib3.poolmanager", line 265, in proxy_from_url
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.packages.urllib3.poolmanager", line 210, in __init__
  File "/Users/jenkins/workspace/build-deis-cli-installer-darwin/src/github.com/deis/deis/client/build/deis/out00-PYZ.pyz/requests.packages.urllib3.util.url", line 185, in parse_url
requests.packages.urllib3.exceptions.LocationParseError: Failed to parse: proxyip:port
```
Here is my setup:

* Vagrant: 1.7.2
* Mac OS 10.10 (Yosemite)
* deis --version 1.3.1

...and confirming...

* http_proxy=http://proxyip:port and https_proxy=http://proxyip:port are set
* deisctl list shows 28 units loaded, active and running

Any suggestions?

thx, G

**Comment by carmstrong** [comment]
Do you need to use a proxy? `deis.local3.deisapp.com` resolves to local VM IPs, so you're essentially taking to localhost over a bridge. 

I'm not sure why the proxy environment variables aren't working - I'd try maybe re-exporting them with quotes (i.e. `export http_proxy="http://1.2.3.4:1234"`).

**Comment by graemenelson** [comment]
I ran into the same issue trying to get up and running with Vagrant.  I ended up just doing:

    export http_proxy=""
    export https_proxy=""

Then it seemed to work. Thanks @carmstrong for pointing out "Do you need to use a proxy?".  I am not sure if this step could be skipped for the Vagrant setup, my guess would be yes but I just started using Deis yesterday.  If it can be left off, than a Vagrant note would be nice.

**Comment by carmstrong** [comment]
> If it can be left off, than a Vagrant note would be nice.

Not sure I follow, @graemenelson -- what made you think you need to use a proxy? Let us know how the documentation could be updated, and we'll certainly do so.

**Comment by graemenelson** [comment]
I was following line by line in the documentation, and I was unclear if I needed proxy support so I just copied/pasted line for line on this page, in particular "Proxy Support":

http://docs.deis.io/en/latest/using_deis/install-client/

Through out the documents there are notes for Vagrant, but not in the "Proxy Support" section.

**Comment by carmstrong** [comment]
> I was unclear if I needed proxy support

I just opened #3169 which hopefully clarifies this. Thanks for the feedback!

**Comment by bacongobbler** [comment]
ping @gpolyn, were you able to resolve this?

**Comment by carmstrong** [comment]
> ping @gpolyn, were you able to resolve this?

There's nothing to resolve - he was unclear about needing proxy support. This will be closed with #3169.
